pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -208,17 +208,17 @@ func (s *DiscoveryServer) ClearCacheFunc() func() {
 			// TODO: metric !!
 			return
 		}
+		s.env.PushContext = push
+		versionLocal := time.Now().Format(time.RFC3339)
 		initContextTime := time.Since(t0)
+		adsLog.Infof("InitContext %v for push takes %s", versionLocal, initContextTime)
```

Comment:
Where did you implement `version` print should be lock protected?

Reply:
I should update, I uses a local var instead.